agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp

```diff
@@ -1268,51 +1261,101 @@ bool backhaul_manager::send_autoconfig_search_message(std::shared_ptr<sRadioInfo
 bool backhaul_manager::send_slave_ap_metric_query_message(uint16_t mid,
                                                           const std::vector<sMacAddr> &bssid_list)
 {
-    bool ret = false;
+    // save mid and clear previous query
+    m_expected_ap_metrics_response.reset_to_new_mid(mid);
+
+    // compare two mac addresses using thier hash values
+    auto mac_comp = [](const sMacAddr &addr1, const sMacAddr &addr2) -> bool {
+        return std::hash<sMacAddr>()(addr1) < std::hash<sMacAddr>()(addr2);
+    };
+
+    // sort the given bssid list
+    std::vector<sMacAddr> given_bssid_list = bssid_list;
+    std::sort(given_bssid_list.begin(), given_bssid_list.end(), mac_comp);
+
+    // debug
```

Comment:
Not sure if this PR should be reviewed or not, as there's a lot of DEBUG code in here...

Reply:
yes, my bad.